### PR TITLE
feat(Pagination): replace toggleTemplateProps with paginationToggleTe…

### PR DESCRIPTION
**Pull request by gitdallas**
closes: #147

### CODE REVIEW

**Review by thatblindgeye — APPROVED** [review]
Looks good! Only nit is using `rename` vs `remove`, but not a blocker.

**Comment by gitdallas** [comment]
> Looks good! Only nit is using `rename` vs `remove`, but not a blocker.

Gonna blame @wise-king-sullyman's generator 😜

**Comment by wise-king-sullyman** [comment]
I mean, the old prop _was_ removed was it not 😂 

Really though I'd be on board with standardizing on calling a rename mod a `rename`, and an actual removal a `remove`, it just seemed that we were just naming the rules `remove` for either case so far from what I could see so I went with that.

**Comment by gitdallas** [comment]
@wise-king-sullyman I suppose the generator could check if the replacement is empty str to determine and do both ways

**Comment by wise-king-sullyman** [comment]
Yeah I'm still planning to circle back to the generator when I get a free moment, restructure things a bit so that handle a few other common use cases like that in a more straightforward way.

That would definitely be a quicker way to implement replace and remove functionality though. I might throw up a quick PR for that.

**Comment by gitdallas** [comment]
@wise-king-sullyman but can you pr review? 😅

File: generate-replace-prop-v5.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const lowerCasedComponentName = componentName[0].toLowerCase() + componentName.s
 const newRuleName = `${lowerCasedComponentName}-remove-${oldPropName}`
 
 if (!componentName || !oldPropName || !newPropName || !referencePR) {
-  console.log('usage: node generate [componentName] [oldPropName] [newPropName] [referencePR]');
+  console.log('usage: node generate-replace-prop-v5 [componentName] [oldPropName] [newPropName] [referencePR]');
```
Comment (wise-king-sullyman) [inline generate-replace-prop-v5.js:12]: Whoops, nice catch.

**Review by wise-king-sullyman — APPROVED** [review]
Yes

**Pull request merged**